rdmo/core/permissions.py

```diff
@@ -85,3 +85,20 @@ class CanToggleElementCurrentSite(DjangoModelPermissions):
     @log_result
     def has_permission(self, request, view):
         return super().has_permission(request, view)
+
+
+class HasRulesPermission(BasePermission):
```

Comment:
this is for the Upload and Import viewsets

Reply:
Nice! I checked the drf and rules code, because I was wondering if there is something like this already. This just introduces the `permission_required` class attribute in the same way as the regular django views, right? I would rename the class to `HasPermission`, since also all the other Permissions somehow depend on rules.